diplomna-fe/src/components/NavBar.tsx

```diff
@@ -1,4 +1,39 @@
-function NavBar() {
+import { useNavigate } from "react-router-dom";
+import logo from "../images/logo-no-background.png"
+import { useState } from "react";
+
+interface NavBarProps {
+  onSearch: (word: string) => void;
+}
+
+function NavBar({onSearch}: NavBarProps) 
+{
+
+    const navigate = useNavigate();
+    const [searchTerm, setSearchTerm] = useState("");
+
+    const handleLogOut = () =>
+    {
+      sessionStorage.removeItem('authToken');
+      navigate("/login");
```

Comment:
I would move these routes to constant variables because if you change the route name you will have to change it in every file